manifests/4.7/cluster-kube-descheduler-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -49,29 +49,23 @@ spec:
       * Node failure requires pods to be moved.
       * New nodes are added to clusters.
 
-      ## Descheduler Strategies
+      ## Descheduler Profiles
 
-      Once the operator is installed, you can configure one or more strategies to identify pods to evict. The scheduler will schedule the replacement of the evicted pods.
+      Once the operator is installed, you can configure one or more profiles to identify pods to evict. The scheduler will schedule the replacement of the evicted pods.
 
-      The following strategies are supported:
-
-      * `LowNodeUtilization` - This strategy finds nodes that are underutilized and evicts pods, if possible, from other nodes in the hope that recreation of evicted pods will be scheduled on these underutilized nodes.
-      * `RemoveDuplicates` - This strategy ensures that there is only one pod associated with a ReplicaSet, ReplicationController, Deployment, or Job running on same node. If there are more, then those duplicate pods are evicted for better spreading of pods in a cluster.
-      * `RemovePodsViolatingInterPodAntiAffinity` - This strategy ensures that pods violating inter-pod anti-affinity are removed from nodes.
-      * `RemovePodsViolatingNodeAffinity` - This strategy ensures that pods violating node affinity are removed from nodes.
-      * `RemovePodsViolatingNodeTaints` - This strategy ensures that pods violating `NoSchedule` taints on nodes are removed.
-      * `RemovePodsHavingTooManyRestarts` - This strategy ensures that pods having too many restarts are removed from nodes.
-      * `PodLifeTime` - This strategy evicts pods that are older than a specified time
-
-      These strategies are documented in detail in the [descheduler README](https://github.com/kubernetes-sigs/descheduler/#policy-and-strategies).
+      The following profiles are available:
+      
+      * AffinityAndTaints
+      * TopologyAndDuplicates
+      * LifecycleAndUtilization
 
+      These profiles are documented in detail in the [descheduler README](https://github.com/kubernetes-sigs/descheduler/#profiles).
+      
       ## Additional Parameters
-
+      
       In addition to the strategies, the following parameters can be configured:
```

Comment:
`In addition to the strategies` -> `In addition to the profiles`

Reply:
@ingvagabund thanks, fixed.